kurtosis-module/impl/participant_network/prelaunch_data_generator/prelaunch_data_generator_launcher/prelaunch_data_generator_launcher.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	image = "ethpandaops/ethereum-genesis-generator:latest"
+	image = "ethpandaops/ethereum-genesis-generator:v1.0.2"
```

Comment:
Oh wow, I thought I did that in my previous PR. Thanks for fixing!

Reply:
no problem :)